routers/events.js

```diff
@@ -0,0 +1,175 @@
+const { Router } = require("express");
+const auth = require("../auth/middleware");
+const Event = require("../models").event;
+const Team = require("../models").team;
+const TeamMate = require("../models").teamMate;
+const Rsvp = require("../models").rsvp;
+const Comment = require("../models").comment;
+const User = require("../models").user;
+
+const router = new Router();
+
+router.get("/", async (req, res) => {
+  const limit = req.query.limit || 10;
+  const offset = req.query.offset || 0;
+  const events = await Event.findAndCountAll({
+    limit,
+    offset,
+    include: ["attending", "user"],
```

Comment:
this works? i normally do it like:
```
include: [{ model: User, as "attending" }]
```

Reply:
it works as it is including all users attending (through rsvp model) and the user who created the event.
i had to try a lot of thing but this worked.